engines/bastion_katello/app/assets/javascripts/bastion_katello/content-hosts/details/views/content-host-details.html

```diff
@@ -12,9 +12,14 @@ <h2 class="fl" translate>Content Host {{ contentHost.name }}</h2>
 
     <div class="fr">
 
-      <div bst-modal="unregisterContentHost(contentHost)" model="contentHost"
-           modal-header='Unregister Content Host "{{contentHost.name}}"?'
-           modal-body='Are you sure you want to unregister content host "{{ contentHost.name }}"?'></div>
+      <div bst-modal="unregisterContentHost(contentHost)" model="contentHost">
```

Comment:
The attribute 'model' is no longer needed since the modal contents are transcluded.